index.html

```diff
@@ -285,6 +285,7 @@ <h2 class="h2-desktop">Let’s chat!</h2>
         </p>
       </div>
       <form
+        novalidate
         action="https://formspree.io/f/mlezyjrr"
         method="post"
         onsubmit="return ErrorMessage()"
```

Comment:
- [ ] [OPTIONAL] For JavaScript best practices kindly add event Lintner on the JavaScript file instead of the Html this is recommended